client/src/main/java/com/aliyun/emr/rss/client/ShuffleClientImpl.java

```diff
@@ -854,9 +855,15 @@ public RssInputStream readPartition(String applicationId, int shuffleId, int red
 
         if (response != null && response.status() == StatusCode.Success) {
```

Comment:
In which case response will be null?  @waitinfuture

Reply:
I think it will never be null.